Modix.Services/BehaviorBase.cs

```diff
@@ -28,14 +28,14 @@ public BehaviorBase(IServiceProvider serviceProvider)
         public async Task StartAsync()
         {
             await OnStartingAsync();
-            IsStarted = true;
+            IsRunning = true;
         }
 
         /// <inheritdoc />
         public async Task StopAsync()
         {
-            await OnStoppingAsync();
-            IsStarted = false;
+            IsRunning = false;
+            await OnStoppedAsync();
```

Comment:
I don't know how serious the implications are, but reordering this means that `IsRunning` could incorrectly return `false` if `OnStoppedAsync()` throws before completing.

Reply:
Hmmm..... THAT is a good point. It doesn't worry me too much, though. For one, we're not actually calling this, ever. For two, if we did, this would only be called during bot shutdown, in which case the exception would just bubble up to the top of the call stack and kill the bot a little early.

Reply:
The `StopAsync()` method? `IBehavior` has` StartAsync()` and `StopAsync()` to match the fact that `DiscordSocketClient` has those methods.